blog/_posts/2016-09-10-biojulia2016-mid.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: BioJulia 2016 - The First Report
+author: Kenta Sato
+---
+
+We are pleased to announce releasing
+[Bio.jl](https://github.com/BioJulia/Bio.jl) 0.4, a minor release but including
+significant functionality improvements as I promised in [the previous blog
+post](http://julialang.org/blog/2016/04/biojulia2016).
+
+The following features are added since the post:
+
+* Online sequence search algorithms.
+* Sequence data structure for reference genomes.
+* Data reader and writer for the .2bit file format.
+* Data reader and writer for the SAM and BAM file formats.
+* Sequence demultiplexing tool.
+* Package to handle BGZF files.
+
+And many other miscellaneous performance and usability!  Tutorial notebooks are
+available at <https://github.com/BioJulia/BioTutorials>.  Here I briefly
+introduce you to these new features one by one.
+
+
+## Online sequence search algorithms
+
+Sequence search is definitely an indispensable tool in sequence analysis.
+Algorithms added in this project term are three types: exact, approximate, and
```

Comment:
My phrasing would be "Since the last post, I have added exact, approximate and regex search algorithms"